core-api/src/main/java/com/optimizely/ab/config/audience/AudienceIdCondition.java

```diff
@@ -26,24 +27,15 @@
 /**
  * The AudienceIdCondition is a holder for the audience id in
  * {@link com.optimizely.ab.config.Experiment#audienceConditions auienceConditions}.  The AudienceId is later
- * resoloved to Audience before evaluation.  If you do not resolve the AudienceIdCondition before evalutating, the
- * condition will fail.  AudienceIdCondtions are resolved using
- * {@link com.optimizely.ab.internal.ConditionUtils#resolveAudienceIdConditions(com.optimizely.ab.config.ProjectConfig, Condition)}
+ * resolved to Audience before evaluation.  If you do not resolve the AudienceIdCondition before evaluating, the
+ * condition will fail.  AudienceIdConditions are resolved when the ProjectConfig is passed into evaluate.
```

Comment:
I think this JSDoc might be unnecessary now, but if we keep it, I would change:

* "resolved to Audience before evaluation" -> "resolved to Audience _during_ evaluation"
* "If you do not resolve the AudienceIdCondition before evaluating, the condition will fail." -> "If you do not pass the ProjectConfig that is necessary for audience resolution, evaluation will fail."

or something like that.

Reply:
fixed